### ISSUE TITLE
xUNC KeyError in L2 spectralReflectance with zang glint correction

### ISSUE THREAD

**Issue by raphidoc** [nasa/HyperCP#234]
xUNC seems to miss the last wavelength here:
https://github.com/nasa/HyperCP/blob/8ed6754e0949641121e19bb24a413fb8e921c742/Source/ProcessL2.py#L522-L538

Comments in the code indicate that this error has already occurred. For me, it happened with the HOCR class-based regime: 

``` python
 File "/home/raphael/PycharmProjects/HyperCP/Source/ProcessL2.py", line 525, in spectralReflectance
    newRhoUNCHyper.columns[k].append(xUNC[f'rhoUNC_{sensor}'][k])
                                     ~~~~~~~~~~~~~~~~~~~~~~~~^^^
KeyError: '802.1'
```

**Comment by raphidoc** [comment]
Hi @ARamsay17, I've uploaded the test data on the box (Data > issue234).

**Comment by ARamsay17** [comment]
@raphidoc Thanks for the data and my apologies it took so long for me to get around to fixing this. I have found the issue and the bugfix will be included in my next PR.

**Comment by raphidoc** [comment]
Thank you @ARamsay17 ! I'll reprocess my data with Z17 soon :)

**Comment by ARamsay17** [comment]
@raphidoc The PR has now been merged and you should be able to continue your reprocessing. Please let us know if you encounter any more issues.

**Comment by oceancolorcoder** [comment]
Resolved with #263. Will be available shortly with the next merge to master (or you can use dev in the meantime).
